### PR TITLE
lastcomm: add new page (#3086)

### PR DESCRIPTION
**Pull request by lmazardo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/lastcomm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# lastcomm
+
+> Show last commands executed (from acct - GNU Accounting utilities).
```
Comment (schneiderl) [inline pages/linux/lastcomm.md]: Is the part "(from acct - GNU Accounting utilities)" really necessary? 

The idea of tldr is to keep things simple, providing a simple description with practical examples. In my opinion, this information is not necessary for a tldr page.

File: pages/linux/lastcomm.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://manpages.debian.org/stable/acct/lastcomm.1.en.html>.
+
+- Print informations about all of the commands in the acct (record file):
+
+`lastcomm`
+
+- Display commands executed by {{user}}:
```
Comment (schneiderl) [inline pages/linux/lastcomm.md]: It is not common (at least I didn't find on a quick search) to use tokens on command descriptions on tldr pages.

I believe we can just remove the curly brackets from the descriptions here and in other descriptions of this page.

File: pages/linux/lastcomm.md (hunk 1 above)
Comment (lmazardo) [reply pages/linux/lastcomm.md]: removed.

File: pages/linux/lastcomm.md (hunk 2 above)
Comment (lmazardo) [reply pages/linux/lastcomm.md]: done. Thanks for the feedback.

File: pages/linux/lastcomm.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display commands executed by user:
+
+`lastcomm --user {{user}}`
+
+- Display informations about command executed on the system:
+
+`lastcomm --command {{command}}`
+
+- Display informations about terminal_name activities executed on the system:
```
Comment (agnivade) [inline pages/linux/lastcomm.md]: Display information about commands executed on a given terminal

Comment (lmazardo) [reply pages/linux/lastcomm.md]: done in commit 8d8e56f3a2054eeb9191c724e23142ea36d300a1

File: pages/linux/lastcomm.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Print informations about all of the commands in the acct (record file):
+
+`lastcomm`
+
+- Display commands executed by user:
+
+`lastcomm --user {{user}}`
+
+- Display informations about command executed on the system:
```
[the remark this answers is not in the document]
Comment (lmazardo) [reply pages/linux/lastcomm.md]: done in commit 8d8e56f3a2054eeb9191c724e23142ea36d300a1

File: pages/linux/lastcomm.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# lastcomm
+
+> Show last commands executed.
+> More information: <https://manpages.debian.org/stable/acct/lastcomm.1.en.html>.
+
+- Print informations about all of the commands in the acct (record file):
+
+`lastcomm`
+
+- Display commands executed by user:
```
[the remark this answers is not in the document]
Comment (lmazardo) [reply pages/linux/lastcomm.md]: done in commit 8d8e56f3a2054eeb9191c724e23142ea36d300a1

**Pull request merged**